alpha-build-machinery/make/examples/multiple-binaries/Makefile.test

```diff
@@ -8,12 +8,12 @@ all:
 	# test version is set correctly when linking
 	#   majorFromGit, minorFromGit are deprecated upstream and set to empty value
 	#   we avoid comparing time to avoid flakes
-	diff <( ./oc | sed '$$d' ) <( \
+	#   we avoid comparing git tree state
+	diff <( ./oc | grep -v -e 'clean' -e 'dirty' | sed '$$d' ) <( \
```

Comment:
oc?

Reply:
ambiguous https://github.com/openshift/library-go/tree/963a391ce8fa3eeed242845a9eee88b9ce24b798/alpha-build-machinery/make/examples/multiple-binaries/cmd/oc

Reply:
it's just an example, right? So nvmd.

Reply:
but why the change?

Reply:
clean vs. dirty has pretty much always been the issue - if you run `make update` before you commit and after, you get different results. also if part of the update is writing down the updated log files this is pretty much a cycle